README.md

```diff
@@ -111,7 +111,31 @@ import About from 'pages/About'
 
 ## Config
 
-### TCR (Optional)
+Default app configs can be found on [the default config file](./config-default.yaml)
+
+We recommend against editing this file directly, though.
+
+If you wish to replace any default config, create a file named `config` inside the [`custom` folder](./custom).
```

Comment:
Probably we can create already the custom folder and leave it empty, just leave an empty `.gitkeep` file

Reply:
I did that at some point but I guess it was lost in a merge/rebase. Will add it.